docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -2,7 +2,9 @@
 [chapter, role="xpack"]
 = Host isolation
 
-Host isolation allows you to isolate hosts from your network, blocking communication with other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. Isolated hosts, however, can still send data to {es} and {kib}.
+Host isolation allows you to isolate hosts from your network, blocking communication with other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. 
+
+Isolated hosts, however, can still send data to {es} and {kib}. You can also create <<host-isolation-exceptions, host isolation exceptions>> for specific IP addresses that isolated hosts are still allowed to communicate with after they've been blocked from the rest of your network.
```

Comment:
I like it, more concise. This phrasing appears in a few other spots so I'll edit those, too. Thanks!

```suggestion
Isolated hosts, however, can still send data to {es} and {kib}. You can also create <<host-isolation-exceptions, host isolation exceptions>> for specific IP addresses that isolated hosts are still allowed to communicate with, even when blocked from the rest of your network.
```